src/map.js

```diff
@@ -163,14 +164,19 @@ export default class GpxMap {
         line.addTo(this.map);
 
         this.tracks.push(line);
+    }
 
+    recenter() {
```

Comment:
Only update the user's location if they haven't already done so themselves. This means that, if a user has only added tracks, then the view will zoom out to fit any newly added tracks. However, if the user has also scrolled the map, then this scrolled view will remain even when new trails are added.

Reply:
Could you add this info (perhaps condensed slightly) as comments in the code?

Reply:
443e466e3513891ef4bff9c5029d4e63c64d09ea